appinventor/appengine/src/com/google/appinventor/client/Ode.java

```diff
@@ -2431,7 +2431,7 @@ public boolean isTutorialVisible() {
   }
 
   public void setTutorialURL(String newURL) {
-    if (newURL.isEmpty() || (!newURL.startsWith("http://appinventor.mit.edu/")
+    if (newURL.isEmpty() || (!newURL.startsWith("https://appinventor.mit.edu/")
```

Comment:
```suggestion
    if (newURL.isEmpty() || (!newURL.startsWith("http://appinventor.mit.edu/")
            && !newURL.startsWith("https://appinventor.mit.edu/"))) {
```

@ewpatton Will this work?